src/node/destination.rs

```diff
@@ -59,32 +50,51 @@ impl AudioNode for AudioDestinationNode {
         1
     }
 
-    fn channel_count_mode(&self) -> ChannelCountMode {
-        ChannelCountMode::Explicit
-    }
-
-    fn channel_interpretation(&self) -> ChannelInterpretation {
-        ChannelInterpretation::Speakers
+    fn set_channel_count(&self, v: usize) {
+        if self.registration.context().offline() && v != self.max_channels_count() as usize {
+            panic!("NotSupportedError: not allowed to change OfflineAudioContext destination channel count");
+        }
+        if v > self.max_channels_count() as usize {
+            panic!(
+                "IndexSizeError: channel count cannot be greater than maxChannelCount ({})",
+                self.max_channels_count()
+            );
+        }
+        AudioNode::set_channel_count(self, v);
     }
-
-    fn channel_count(&self) -> usize {
```

Comment:
The AudioDestinationNode can now change its desired number of output channels (only online AudioContext, not offline)